### PR TITLE
Add `Client.get_message`

### PR DESCRIPTION
**Pull request by Dorukyum**
## Checklist
- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting, examples, ...)

### CODE REVIEW

**Comment by Dorukyum** [comment]
> Intents documentation could be updated to add this method to 'affected attributes'

This is only affected by the `message` intent which isn't privileged and so not in that documentation

**Comment by plun1331** [comment]
> > Intents documentation could be updated to add this method to 'affected attributes'
> 
> This is only affected by the `message` intent which isn't privileged and so not in that documentation

The other intent attributes note the methods/attributes that rely on them, such as `guilds`

**Comment by Dorukyum** [comment]
> > > Intents documentation could be updated to add this method to 'affected attributes'
> > 
> > 
> > This is only affected by the `message` intent which isn't privileged and so not in that documentation
> 
> The other intent attributes note the methods/attributes that rely on them, such as `guilds`

I see what you mean now, I've updated it.

**Review by ChickenDevs — DISMISSED** [review]

**Review by RGBCube — APPROVED** [review]

File: discord/flags.py [hunk 1]
```diff
@@ -741,6 +741,7 @@ def messages(self):
 
         - :class:`Message`
         - :attr:`Client.cached_messages`
+        - :meth:`Client.get_message`
```
[the remark this answers is not in the document]
Comment (Dorukyum) [reply discord/flags.py:744]: Updated.

**Review by krittick — APPROVED** [review]

**Review by Middledot — APPROVED** [review]

**Pull request merged**